pageserver/src/walreceiver/walreceiver_connection.rs

```diff
@@ -151,14 +155,14 @@ pub async fn handle_walreceiver_connection(
                 waldecoder.feed_bytes(data);
 
                 while let Some((lsn, recdata)) = waldecoder.poll_decode()? {
-                    let _enter = info_span!("processing record", lsn = %lsn).entered();
+                    // let _enter = info_span!("processing record", lsn = %lsn).entered();
```

Comment:
I see this as free performance improvement. Can revert the change in this PR if needed